docs/content/reference/configuration/agent.md

```diff
@@ -1586,8 +1586,9 @@ Aperture Controller.
 
 <!-- vale on -->
 
-Address to gRPC or HTTP(s) server listening in agent service. To use HTTP
-protocol, the address must start with `http(s)://`.
+Address to gRPC or HTTP(s) server listening in agent service. For connecting to
+FluxNinja Cloud Controller, the `endpoint` should be a `grpc/http2` address. For
+self-hosted controller, the HTTP protocol address must start with `http(s)://`.
```

Comment:
It's "can" rather than "must"

Reply:
Address in [3bbfb09](https://github.com/fluxninja/aperture/pull/2479/commits/3bbfb09b2bc5e67d4cf95646a2176606076f7437).